packages/account/src/Components/trading-assessment/trading-assessment-dropdown.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { Field } from 'formik';
+import { DesktopWrapper, Dropdown, MobileWrapper, Text, SelectNative } from '@deriv/components';
+import { localize } from '@deriv/translations';
+
+const TradingAssessmentDropdown = ({ item_list, onChange, values, setFieldValue, setEnableNextSection }) => {
+    React.useEffect(() => {
+        checkIfAllFieldsFilled();
+    }, [values]);
+
+    const checkIfAllFieldsFilled = () => {
+        setEnableNextSection(
+            values.cfd_experience &&
+                values.cfd_frequency &&
+                values.trading_experience_financial_instruments &&
+                values.trading_frequency_financial_instruments
+        );
+    };
+
+    return (
+        <div className='trading-assessment__wrapper__dropdown'>
+            {item_list.map((question, index) => (
+                <Field name={question.form_control} key={index}>
+                    {() => {
+                        return (
+                            <React.Fragment>
+                                <DesktopWrapper>
+                                    <Dropdown
+                                        classNameDisplay='trading-assessment__wrapper__dropdown--mobile--display'
+                                        is_align_text_left
+                                        name={question?.question_text}
+                                        placeholder={question?.question_text}
+                                        list={question?.answer_options}
+                                        onChange={e => onChange(e, question.form_control, setFieldValue)}
+                                        value={values[question.form_control]}
+                                    />
+                                </DesktopWrapper>
+                                <MobileWrapper>
+                                    <Text as='h1' color='prominent' weight='bold' size='xs'>
+                                        {question?.question_text}
+                                    </Text>
+                                    <SelectNative
+                                        placeholder={localize('Please select')}
+                                        label={question?.answer_options[0].text}
+                                        name={question?.question_text}
+                                        list_items={question?.answer_options}
+                                        onChange={e => {
+                                            onChange(e, question.form_control, setFieldValue);
+                                        }}
+                                        value={values[question.form_control]}
+                                        hide_placeholder={true}
```

Comment:
`hide_placeholder` has a default value of `false` hence the reason why I specify to `true` when passing the props.

Reply:
I think just existing of this field means that it is true, and if you don’t pass it to the component it will use default value. So we don’t need to use {true} explicitly.
please correct me